### DISCUSSION TITLE
Set Sort Order of Pinned Collections?

### DISCUSSION THREAD

**Discussion by TheLunarMedia** [meisnate12/Plex-Meta-Manager#2298] [Q&A]
I see that I am able to make collections visible on Home recommended tab using **visible_home**
I also see I can schedule this with  **visible_home: range(12/01-12/31)**

QUESTION: How do I set the sort order IN THE HOME RECOMMENDED TAB once it pinned added? 

ISSUE: I have a seasonal collection that adds/deletes collections based on date schedule (essentially the same as the default seasonal collection). I want the holiday section to always be at the top of the recommended tab whenever it is scheduled. THE PROBLEM is that when it is added, I have to manually sort to bring it to the top within plex. Then when it is no longer scheduled (and automatically deleted), the next time it comes back I have to manually sort it again. Is there any way to have it behave like how plex has their seasonal collection always showing in that tab so it remembers the sort position, or am I able to specify the sort position of that section via Kometa code?

**Comment by chazlarson** [comment]
This is not exposed to the Plex API, so Kometa cannot set the position.

[or if it is exposed, Kometa has not wired it up]

**Reply by TheLunarMedia** [reply]
Thank you!

**Reply by TheLunarMedia** [reply]
I thought of a workaround but not sure if I can do that either... Is there a way to determine the sort title based on a schedule? That way I could have all the holiday collections at all times but sort them to the bottom of my collection section when not scheduled...

**Reply by chazlarson** [reply]
There is not a way to set just one attribute of a collection on a schedule.

**Reply by chazlarson** [reply]
I was under the impression that once you set the position of "My Cool Collection" it would appear in the same place when it reappeared.  Perhaps Plex has changed?

You may want to take this to the discord for more eyes.

**Reply by KellyNewman87** [reply]
It's available in the Plex API now:
https://developer.plex.tv/pms/#tag/Hubs/operation/hubsSectionsSectionManagePutMove
